pkg/karmadactl/join.go

```diff
@@ -201,8 +194,8 @@ func JoinCluster(controlPlaneRestConfig, clusterConfig *rest.Config, clusterName
 			Name:      clusterName,
 		},
 		Data: map[string][]byte{
-			caDataKey: clusterSecret.Data["ca.crt"], // TODO(RainbowMango): change ca bundle key to 'ca.crt'.
-			tokenKey:  clusterSecret.Data[tokenKey],
+			clusterv1alpha1.SecretCADataKey: clusterSecret.Data["ca.crt"], // TODO(RainbowMango): change ca bundle key to 'ca.crt'.
```

Comment:
```suggestion
			clusterv1alpha1.SecretCADataKey: clusterSecret.Data["ca.crt"], 
```
Please help remove the `TODO` as it's hard and not worth making such an incompatible change now.

Reply:
Agree.